### DISCUSSION TITLE
AUX Connector?

### DISCUSSION THREAD

**Discussion by jnz86** [blackmagic-debug/blackmagic#1928] [Q&A]
I haven't been able to find any information on the AUX connector on the v2.3.

What I need is at minimum two GPIOs that I can use to drive a mosfet gate and an LED. 

I am making an assumption that the process would be that in the BMP driver there will be a BPM-internal command to set/reset AUX connector pins. Does any of this exist yet?

It would be nice to know what the other options are. Will there be UART-pass through or SPI or ???

**Comment by dragonmux** [comment]
👋🏼 The aux connector is an internal expansion connector that was not intended for general use - however, it exposes the internal SPI bus which the remote SPI protocol can be used to talk on (chip selects provided by the interface are exposed in the remote protocol), and two button inputs (one in parallel with the user button on the main board).

You could technically abuse the remote protocol to toggle the chip selects, but there are no plans to make any of this available as straight GPIO to the user as it's designed for a specific purpose involving an add-on board for stand-alone usage in a factory setting.

There are no UARTs pinned out to the connector.
